en/snap/platform/overview.md

```diff
@@ -383,6 +383,28 @@ The **detailed history will be displayed on Shopify admin panel's order details
 Here is an example of 1 order that has 1 earlier payment id which is unpaid, and then 1 other payment id which is successfully paid, as observed from Shopify admin panel's order details timeline.
 ![Timeline 1 Order 2x Payments](./../../../asset/image/shopify-new-24-1-order-2x-payment.png ':size=400')
 
+##### Payment Method Icons on Shopify Payment Setting Page
+On the Shopify payment setting page, you will find there are payment method icons, some of you maybe think that the icons represent the payment methods on Midtrans, but it is not true, it does not represent the payment methods on Midtrans.
+![Payment icons](./../../../asset/image/shopify-new-25-payment-icons.png ':size=400')<br>
+That is just an icons selection screen on the Shopify side and will show on the payment method on the checkout page. It does not affect the actual payment methods on Midtrans.
+![Payment icons](./../../../asset/image/shopify-new-25-payment-icons-on-checkout.png ':size=400')<br>
+If you want to enable specific payment methods and affect on the Snap payment page, you can enable it from [Payment App account settings.](#payment-app-account-settings)
+
+##### Test Order with Enabling Test Mode is Not Charged
+Test order is helpful for you to make sure that the checkout process and settings for order processing, payment, inventory, shipping, email notifications, etc are all correct. You can place a test order by [simulating a transaction through test mode](#test-transaction-with-sandbox-mode)
+
+**You are not charged for a test order**, all orders made within this mode are not "real", and do not require "real payment/fund", so you are also **safe to ignore the payment for this order**.
+
```

Comment:
Charged what exactly?

Reply:
> all orders made within this mode are not "real", and do not require "real payment/fund"

But `so you are also **safe to ignore the payment for this order**.` is not exactly related to the previous sentence. Also clarify what does exactly `safe to ignore the payment` means for merchant? do you mean order should be safe to not actually fulfilled?

Reply:
> **You are not charged for a test order**,

How can you (as a Midtrans) be so sure that some other party (Shopify) will not charge? You should either:
- Word it carefully: Shopify should not charge you any payment processing fee for "test order" payments. 
- Or back it up with resource/link to Shopify explanation.